acceptance-tests/src/test/resources/enclave_key1.pub

```diff
@@ -0,0 +1 @@
+A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=
```

Comment:
Do we need to start these as files, or could we just make random strings - i.e. they don't need to actually contain real keys... (do they?!)

Reply:
They are real test public keys I created. But since we aren't using Orion now they could be just random strings.
Pantheon requires the key as a file and that file must be readable by docker. The default /var/tmp directory Java uses on Mac isn't accessible by Docker by default. So files work out a little bit nicer.